app/src/main/res/menu/stylist_home_bottom_navigation.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
WARNING: [UnusedResources] The resource `R.menu.stylist_home_bottom_navigation` appears to be unused
 Unused resources make applications larger and slow down builds.